aspnetcore/security/cookie-sharing.md

```diff
@@ -40,19 +40,34 @@ In `Startup.ConfigureServices`:
 
 ```csharp
 services.AddDataProtection()
-    .PersistKeysToFileSystem({PATH TO COMMON KEY RING FOLDER})
+    .PersistKeysToFileSystem("{PATH TO COMMON KEY RING FOLDER}")
     .SetApplicationName("SharedCookieApp");
 
 services.ConfigureApplicationCookie(options => {
     options.Cookie.Name = ".AspNet.SharedCookie";
 });
 ```
 
+### Custom authentication builder
+
+Custom authentication builder must set the `Cookie.Path` for single sign on apps. The following code is added to `Startup.ConfigureServices`:
```

Comment:
This title and description don't make any sense. What is "Custom authentication builder" and why must Path be set?

```
### Cookie paths

Authentication cookies use the HttpRequest.PathBase as their default Cookie.Path. If your cookie needs to be shared across different base paths then Path must be overridden. The above example can be modified as show here:
```